src/goto-symex/symex_assign.cpp

```diff
@@ -231,10 +232,108 @@ void goto_symext::symex_assign_symbol(
     tmp_ssa_rhs.swap(ssa_rhs);
   }
 
+  // TODO: done via rename
+  // field_sensitivityt::apply(ns, ssa_rhs, false);
+
   state.rename(ssa_rhs, ns);
+
+  ssa_exprt lhs_mod = lhs;
+
+  if(
+    ssa_rhs.id() == ID_byte_update_little_endian ||
+    ssa_rhs.id() == ID_byte_update_big_endian)
+  {
+    byte_update_exprt &bu = to_byte_update_expr(ssa_rhs);
```

Comment:
avoid abbreviated names, `byte_update` would be better :snake:

Reply:
Done.